src/app/modules/form/form.utils.ts

```diff
@@ -100,6 +101,18 @@ export const getCollabEmailsWithPermission = (
   }, [])
 }
 
+export const getAdminEmails = (
+  form: Partial<IFormHasEmailSchema>,
+): string[] => {
+  if (form.emails === undefined) {
+    return []
+  } else if (typeof form.emails === 'string') {
+    return [form.emails]
+  } else {
+    return form.emails
+  }
```

Comment:
/nit completely stylistic to omit usage of else

```suggestion
  } 
  if (typeof form.emails === 'string') {
    return [form.emails]
  }
  return form.emails
```

Reply:
Resolved!